static/app/views/performance/transactionSummary/transactionOverview/transactionPercentage.tsx

```diff
@@ -0,0 +1,87 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {SectionHeading} from 'sentry/components/charts/styles';
+import Placeholder from 'sentry/components/placeholder';
+import {t, tct} from 'sentry/locale';
+import {Organization} from 'sentry/types';
+import {QueryError} from 'sentry/utils/discover/genericDiscoverQuery';
+import {formatFloat, formatPercentage} from 'sentry/utils/formatters';
+import {SidebarSpacer} from 'sentry/views/performance/transactionSummary/utils';
+
+interface TransactionPercentageProps {
+  error: QueryError | null;
+  isLoading: boolean;
+  organization: Organization;
+  totals: Record<string, number> | null;
+  unfilteredTotals?: Record<string, number> | null;
+}
+
+export function TransactionPercentage({
+  organization,
+  isLoading,
+  error,
+  totals,
+  unfilteredTotals,
+}: TransactionPercentageProps) {
+  const showTPMAsPercentage = organization.features.includes(
+    'performance-metrics-backed-transaction-summary'
+  );
+
+  function getValueFromTotals(field, totalValues, unfilteredTotalValues) {
+    if (totalValues) {
+      if (unfilteredTotalValues) {
+        return tct('[tpm]', {
+          tpm: formatPercentage(totalValues[field] / unfilteredTotalValues[field]),
+        });
+      }
+      return tct('[tpm] tpm', {
+        tpm: formatFloat(totalValues[field], 4),
+      });
+    }
+    return null;
+  }
+
+  return (
+    <Fragment>
+      <SectionHeading>
+        {showTPMAsPercentage ? t('Percentage of Total Transactions') : t('TPM')}
```

Comment:
🧐

Reply:
ah yeah I'm still trying to hide it behind a feature flag for now